src/api/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+// export const getGreetings = async () => {
+//    const response = await fetch("api/v1/greetings");
+
+//    return response.json();
+// };
+
+const getGreetings = async () => axios.get('http://localhost:3000/api/v1/greetings').then((result) => {
+  const greetings = [];
+  if (result.status === 200) {
+    const { data } = result;
+    // eslint-disable-next-line no-restricted-syntax
+    data.forEach((item) => {
+      greetings.push(item.greeting);
+    });
+  }
+  return greetings;
+});
```

Comment:
- [ ] using `async` function without `await` keyword does not give sense. As per the module requirements, I think you should refactor this method to use `async-await` instead of `then`. Kindly fix this issue. :pray:

Reply:
Thanks for the suggestion. I have already refactored it. :+1: